ansible/roles/dev-desktop/tasks/dependencies.yml

```
@@ -46,6 +46,8 @@
       - libatspi2.0-0
       - libelf-dev  # Allows building tools that link against libelf
       - emacs
+      - lld
+      - llvm
```

@albertlarsan68 
```suggestion
      - lldb
```
`llvm` is already installed. Is this what you meant?

---

Yes, thanks!